api/object.rb

```diff
@@ -43,5 +43,16 @@ def validate
     def out_name
       @name.underscore
     end
+
+    # Utility class for Objects
+    module ObjectUtils
+      module_function
+
+      def string_to_object_map?(property)
```

Comment:
The `?` is just notation. It has no other meaning.

As far as I can tell, if a function is returning a boolean, you put a `?` at the end of the function because it's like you're asking a question.

Reply:
Yeah, if you do `is_string_to_object_map` rubocop will complain